src/app/elastic/dsl.py

```diff
@@ -17,6 +17,7 @@ def __new__(cls, path: str, field_type: ElasticFieldType):
         obj._value_ = path
         obj.path = path
         obj.field_type = field_type
+        obj.keyword = f"{path}.keyword"
```

Comment:
I would have defined this as property on the Enum to enable autocompletion. Either way :+1:

Reply:
Yes! Done